.github/workflows/codeql.yml

```
@@ -1,17 +1,101 @@
+      - uses: it-at-m/lhm_actions/action-templates/actions/action-maven-build@1.0.0
+        with:
+          codeql-language: java-kotlin
+          codeql-buildmode: ${{ env.java-buildmode }}
+          codeql-query: ${{ env.analysis-query }}
+          java-version: ${{ env.java-version }}
+          path: ${{ matrix.build-path }}
+  codeql-javascript-typescript-vue:
+    name: Analyze JavaScript/TypeScript/Vue source files
+    runs-on: ubuntu-latest
+    needs: check-changes
+    if: env.analyze-javascript-typescript-vue && (github.ref_name == 'main' || needs.check-changes.outputs.javascript-typescript-vue == 'true')
```

actionlint (1.7.4)

context "env" is not allowed here. available contexts are "github", "inputs", "needs", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)